Extension/src/LanguageServer/client.ts

```diff
@@ -1057,7 +1061,7 @@ export class DefaultClient implements Client {
                     this.showPrompt(selectCompiler, true);
                     action = "dismissed";
                 }
-                telemetry.logLanguageServerEvent('compilerNotification', { action });
+                telemetry.logLanguageServerEvent('compilerNotification', { action }, { sender });
```

Comment:
I believe the args should be `{ action, sender: getSenderType(sender) }`

(`getSenderType` is defined in extension.ts, but the function should move to common.ts and be marked with `export` so you can use it here)